packages/core/src/constants.ts

```diff
@@ -26,29 +26,29 @@ export type ChainId = (typeof CHAIN_IDS)[keyof typeof CHAIN_IDS];
 // Base URLs
 ////////////////////////////////////////////////////////////////////////////////
 
-export const RELAYER_URL_ARBITRUM_MAINNET = "mainnet.cluster0.renegade.fi";
+export const RELAYER_URL_ARBITRUM_ONE = "mainnet.cluster0.renegade.fi";
 export const RELAYER_URL_ARBITRUM_SEPOLIA = "testnet.cluster0.renegade.fi";
 
-export const RELAYER_WS_URL_ARBITRUM_MAINNET = "wss://mainnet.cluster0.renegade.fi";
+export const RELAYER_WS_URL_ARBITRUM_ONE = "wss://mainnet.cluster0.renegade.fi";
 export const RELAYER_WS_URL_ARBITRUM_SEPOLIA = "wss://testnet.cluster0.renegade.fi";
 
-export const AUTH_SERVER_URL_ARBITRUM_MAINNET = "https://mainnet.auth-server.renegade.fi:3000";
+export const AUTH_SERVER_URL_ARBITRUM_ONE = "https://mainnet.auth-server.renegade.fi:3000";
 export const AUTH_SERVER_URL_ARBITRUM_SEPOLIA = "https://testnet.auth-server.renegade.fi:3000";
 
-export const PRICE_REPORTER_URL_ARBITRUM_MAINNET = "mainnet.price-reporter.renegade.fi";
+export const PRICE_REPORTER_URL_ARBITRUM_ONE = "mainnet.price-reporter.renegade.fi";
 export const PRICE_REPORTER_URL_ARBITRUM_SEPOLIA = "testnet.price-reporter.renegade.fi";
 
-export const HSE_BASE_URL_ARBITRUM_MAINNET = "https://mainnet.historical-state.renegade.fi:3000";
+export const HSE_BASE_URL_ARBITRUM_ONE = "https://mainnet.historical-state.renegade.fi:3000";
```

Comment:
to be clear, only the following subset of the resources here are chain-dependent:
1. Relayer
2. Auth server

I.e., for the HSE & price reporter, we don't need separate namespacing per-chain, doing so is confusing.

Same goes for the sepolia namespacing, the HSE & price reporter should stick to the `testnet` / `mainnet` naming convention